tests/unittests/OperatorTest.cpp

```diff
@@ -2450,6 +2452,83 @@ TEST_P(InterpOnly, SparseLengthsSum) {
   EXPECT_TRUE(expected.isEqual(result));
 }
 
+/// A helper function to test quantized element-wise ops on the specific
+/// backend.
+void quantizedElementwiseOpsOnSpecificBackend(BackendKind backend, Tensor *lhs,
```

Comment:
what stops these test to be run on all backends, TESTP_P(Operator ... ?

Reply:
I do not think we need this test. TEST_P(Operator, QuantizedArithmeticUnrescaled) tests the OpenCL quantized operations, make sure that QuantizedArithmeticUnrescaled has all arithmetic operations you are interested in.

After you add the Rescale operator support you could change QuantizedArithmetic*Rescaled* test to be Operator also.

Reply:
You are right. I just remove my test.